includes/syndicated-post-ui.php

```diff
@@ -26,12 +28,8 @@ function() {
 			add_filter( 'admin_body_class', __NAMESPACE__ . '\add_linked_class' );
 			add_filter( 'post_row_actions', __NAMESPACE__ . '\remove_quick_edit', 10, 2 );
 
-			$post = isset( $_GET['post'] ) ? get_post( (int) $_GET['post'] ) : false; // @codingStandardsIgnoreLine Nonce not required
-
-			if ( $post && ! \Distributor\Utils\is_using_gutenberg( $post ) ) {
-				add_action( 'do_meta_boxes', __NAMESPACE__ . '\replace_revisions_meta_box', 10, 3 );
-				add_action( 'add_meta_boxes', __NAMESPACE__ . '\add_revisions_meta_box' );
-			}
+			add_action( 'do_meta_boxes', __NAMESPACE__ . '\replace_revisions_meta_box', 10, 3 );
+			add_action( 'add_meta_boxes', __NAMESPACE__ . '\add_revisions_meta_box' );
```

Comment:
The `is_using_gutenberg()` function call is moved to the `do_meta_boxes` hook where it is used. This ensures all the editor functions are defined before attempting to use them. Specifically, in this instance, `use_block_editor_for_post_type()`.


before the core function it uses are defined. In the past it took an unexpected path but mostly worked. With the cut down version of `is_using_gutenberg()` that assumes a block-editor exists this file needs to wait until the editor is defined before using it. Hence, this change